pandas/_libs/lib.pyx

```diff
@@ -2274,7 +2274,7 @@ def to_object_array_tuples(rows: list):
 
     k = 0
     for i in range(n):
-        tmp = len(rows[i])
+        tmp = 1 if is_float(rows[i]) else len(rows[i])
```

Comment:
I presume you're meaning to check if we have `np.nan` ?  If so, let's be more explicit (e.g. use `pandas.isnull`).  You could also use `is_list_like` from `pandas/core/dtypes/inference.py`.

Reply:
Yes, indeed. That's the case I wanted to handle. I fixed it in the next commit.